src/app/pages/news/11.2/06-cron.md

```diff
@@ -1,6 +1,12 @@
 ## CRON expressions {#cron}
 
-Blah cron 
+You can now configure periodical jobs by using <a href="https://en.wikipedia.org/wiki/Cron">CRON</a> expressions.
+CRON expressions give you a wide spectrum of possible configurations. From once every minute up to once every day, week, month, or year. 
+
+- **User Synchronisation**: Define the time the user synchronization is executed with a CRON expression. 
+- **Timer Bean**: Use a CRON expression on the new Start Event Bean `ch.ivyteam.ivy.process.eventstart.beans.TimerBean` to define when your processes are started.
+- **Poller API**: If you write your own Start and Intermediate Event Beans you can use the new <a href="${docBaseUrl}/public-api/ch/ivyteam/ivy/process/beans/IPoller.html">Poller</a> API to define when your Bean should be polled.
```

Comment:
I'd skip the 'conditional' writing style to make this sentence easier to read: e.g. ..

Poller API: The definition of polling intervals in custom Start and Intermediate Events has become much easier with the new fluent poll() APIs.